src/MeadeCommandProcessor.cpp

```diff
@@ -496,6 +496,25 @@ bool gpsAqcuisitionComplete(int &indicator);  // defined in c72_menuHA_GPS.hpp
 //      Returns:
 //        "1" if successfully scheduled
 //
+// :MHR#
+//      Description:
+//        Home RA stepper via Hall sensor
+//      Information:
+//        This attempts to find the hall sensor and to home the RA ring accordingly.
+//      Parameters:
+//        If the last letter 'R' is uppercase the mount will search east first, if it is lowercase it will search west first.
```

Comment:
Using `R` and `r` is a bit misleading because for me R (regardless if it is upper case or lower case) stands for reverse. Also i did not see similar command syntax before here. We should provide a dedicated character for each rotation direction. We could use something like `:MHL#` and `:MHR#`

Reply:
Agreed